src/node/desktop/CMakeLists.txt

```diff
@@ -211,6 +211,10 @@ elseif(LINUX)
 
    install(
       DIRECTORY "${ELECTRON_BUILD_DIR}/out/RStudio-linux-${ELECTRON_ARCH}/"
+      DIRECTORY_PERMISSIONS
+      OWNER_READ OWNER_WRITE OWNER_EXECUTE
+      GROUP_READ GROUP_EXECUTE
+      WORLD_READ WORLD_EXECUTE
       DESTINATION "${RSTUDIO_INSTALL_ELECTRON}"
       USE_SOURCE_PERMISSIONS
```

Comment:
Is this line still needed? I wonder if the issue reflects an issue with the permissions on the directory created here now that webpack etc. has been updated?

Reply:
I'm not totally familiar with `cmake` so I didn't try without the `USE_SOURCE_PERMISSIONS` option. It seemed that only the top-level directory was affected by the change to be user only accessible.

My thought is that it is needed to preserve the file permissions.

Reply:
Makes sense! Thanks; let's keep it.